src/SFA.DAS.Apprenticeships.Web/Views/ChangeOfStartDate/Provider/CheckDetails.cshtml

```diff
@@ -1,13 +1,64 @@
 ﻿@using SFA.DAS.Apprenticeships.Web.Models.ChangeOfStartDate
 @model ProviderChangeOfStartDateModel
 @{
-	ViewData["Title"] = "Check your changes before sending to the employer - Manage apprenticeships";
-	var editPage = $"/provider/{Model.ProviderReferenceNumber}/ChangeOfPrice/{Model.ApprenticeshipHashedId}/edit?cacheKey={Model.CacheKey}";
+    ViewData["Title"] = "Check your changes before sending to the employer - Manage apprenticeships";
+    var editPage = $"/provider/{Model.ProviderReferenceNumber}/ChangeOfStartDate/{Model.ApprenticeshipHashedId}/edit?cacheKey={Model.CacheKey}";
 }
 
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <h1 class="govuk-heading-l">Check your change before sending to the employer</h1>
+        <h2 class="govuk-heading-m">Change you're requesting</h2>
+        <table class="govuk-table">
+            <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+                <th scope="col" class="govuk-table__header app-custom-class govuk-!-width-one-third"/>
+                <th scope="col" class="govuk-table__header app-custom-class">Current</th>
+                <th scope="col" class="govuk-table__header app-custom-class govuk-!-width-one-third">Requested</th>
+                <td class="govuk-table__cell"></td>
+            </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
+                <th scope="row" class="govuk-table__header">Training start date</th>
+                <td class="govuk-table__cell">@Model.OriginalApprenticeshipActualStartDate.GetValueOrDefault().ToString("d MMMM yyyy")</td>
+                <td class="govuk-table__cell">
+                        @Model.ApprenticeshipActualStartDate?.Date.GetValueOrDefault().ToString("d MMMM yyyy")
+                        @if (Model.PreviousAcademicYearEndDate > Model.ApprenticeshipActualStartDate?.Date.GetValueOrDefault() && Model.PreviousAcademicYearHardCloseDate < DateTime.Now)
+                        {
+                            <span style="width:26px; position: relative; float: left; margin:0px 4px 0px 0px">
```

Comment:
Is it worth creating a component for this so its obvious from the code what it is. Something like ExclamationSvgComponent